lib/hl7/header.js

```diff
@@ -3,102 +3,82 @@ var delimiters = require('./delimiters');
 var segment = require('./segment');
 
 function header() {
-
-  this.name = "MSH";
-  this.delimiters = {
-    fieldSeperator: "|",
-    componentSeperator: "^",
-    subcomponentSeperator: "&",
-    escapeCharacter: "\\",
-    repititionCharacter: "~",
-    segmentSeperator: '\r'
-  };
+  this.name = 'MSH';
+  this.delimiters = delimiters;
 
   this.fields = [];
 
   if (arguments.length > 1) {
-
     for (var i = 0; i < arguments.length; i++) {
-
-      if (Array.isArray(arguments[i])) {
-        var fields = new Array();
-        for (var ii = 0; ii < arguments[i].length; ii++) {
-          fields.push(new field(arguments[i][ii]));
-        }
-        this.fields.push(fields);
-      } else {
-        for (var i = 0; i < arguments.length; i++) {
-          this.fields.push(new field(arguments[i]));
-        }
-      }
+      this.fields.push(new field(arguments[i]));
```

Comment:
This didn't work as shown by https://github.com/hitgeek/simple-hl7/pull/104/files#diff-f2fb4042ae9e7bb6bcb9bf1c2d6eb277efb0402f7c36e6988a2fe48810e6bd07R207